addsvc/grpc_binding.go

```diff
@@ -20,15 +20,15 @@ type grpcBinding struct{ server.Endpoint }
 // way to manipulate the RPC context, like headers for HTTP. So we don't have
 // a way to transport e.g. Zipkin IDs with the request. TODO.
 func (b grpcBinding) Add(ctx context.Context, req *pb.AddRequest) (*pb.AddReply, error) {
```

Comment:
Thanks for this :+1: I was doing the opposite and using main repo as `upstream`. This works much better.